src/len_type.rs

```diff
@@ -36,6 +38,16 @@ pub trait Sealed:
     fn into_usize(self) -> usize {
         self.try_into().unwrap()
     }
+
+    /// Converts `LenT` into `Some(usize)`, unless it's `Self::MAX`, where it returns `None`.
+    #[inline]
+    fn option(self) -> Option<usize> {
```

Comment:
Is this still used anywhere?

Reply:
Please rename this while you're at it, since `option` is a meaningless name.

Maybe `to_non_max`?

Reply:
Done.